internal/graphicsdriver/opengl/context_js.go

```diff
@@ -12,6 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:build js
+// +build js
```

Comment:
I don't think we should care this. Wouldn't it be possible for GopherJS to recognize `*_js.go` files?